pom.xml

```diff
@@ -9,12 +9,7 @@
 
     <packaging>jar</packaging>
 
-    <parent>
-        <version>1.0.6</version>
-        <groupId>uk.ac.susx.tag</groupId>
-        <artifactId>tag-dist</artifactId>
-        <relativePath>../tag-dist</relativePath>
-    </parent>
```

Comment:
This is required, why is it removed?

Reply:
I removed it because it is mentioned in the read me file

Reply:
Ah, that's just to get things running locally without having access to the `tag-dist` repo, but those lines of code should remain in this git repo - they are used for deploying the code.

Reply:
Okay added them back